src/views/search/search.jsx

```diff
@@ -96,6 +95,31 @@ var Search = injectIntl(React.createClass({
         }
         return allTab;
     },
+    getProjectBox: function () {
+        var results = <Grid items={this.state.loaded}
```

Comment:
We're starting to move to a different style for this kind of multiline – would you mind changing this slightly to something like:
```jsx
var results = <Grid
    items={this.state.loaded}
    itemType={this.props.tab}
    cards={true}
    showAvatar={true}
    showLoves={false}
    showFavorites={false}
    showViews={false}
/>;
```